conf/test.config

```diff
@@ -46,26 +46,29 @@ params {
 }
 
 process {
-    withName:'.*:FREEC_SOMATIC'{
-        ext.args = {
-            [
-                "sample":[
-                    inputformat: "pileup",
-                    mateorientation: "FR"
-                ],
-                "general":[
-                    bedgraphoutput: "TRUE",
-                    noisydata: "TRUE",
-                    minexpectedgc: "0",
-                    readcountthreshold: "1",
-                    sex: meta.sex,
-                    window: "10",
+
+    if (params.tools && params.tools.split(',').contains('controlfreec')) {
+        withName:'.*:FREEC_SOMATIC'{
```

Comment:
```suggestion
        withName:'FREEC_SOMATIC'{
```

Reply:
I think then we can even get rid of the if clause again, I just wanted to remove the annoying warning selector messages